client/components/dispute-status-chip/index.tsx

```diff
@@ -21,10 +22,24 @@ import type {
 interface Props {
 	status: DisputeStatus | string;
 	dueBy?: CachedDispute[ 'due_by' ] | EvidenceDetails[ 'due_by' ];
+	includeTransactionType?: boolean;
 }
-const DisputeStatusChip: React.FC< Props > = ( { status, dueBy } ) => {
+const DisputeStatusChip: React.FC< Props > = ( {
+	status,
+	dueBy,
+	includeTransactionType,
+} ) => {
 	const mapping = displayStatus[ status ] || {};
-	const message = mapping.message || formatStringValue( status );
+	let message = mapping.message || formatStringValue( status );
+
+	// Statuses starting with warning_ are Inquiries and these are already prefaced with "Inquiry: "
+	if ( includeTransactionType && ! status.startsWith( 'warning' ) ) {
+		message = sprintf(
+			/** translators: %s is the status of the Dispute. */
+			__( 'Disputed: %s', 'woocommerce-payments' ),
+			message
+		);
+	}
```

Comment:
I'm not super confident about this approach, seems a little hacky. Also open to suggestions on a better flag that `includeTransactionType`

Reply:
Can we just pass in the message from above? This thing doesn't need to format the message (much).

Reply:
Unfortunately not as this message is used to map to different statuses and colors

Reply:
Yea to make this component dumb we need to do some brain surgery / lobotomy 🔪

Reply:
The colour / urgency would be an excellent prop to expose 😁